querido_diario/etl/transform_wrapper.py

```diff
@@ -0,0 +1,7 @@
+exit()
```

Comment:
please ignore or reject change... this will be moved to my scratch space. i've uploaded as a mistake